shared-libs/search/src/generate-search-requests.js

```diff
@@ -191,7 +191,7 @@ const sortByLastVisitedDate = () => {
 const makeCombinedParams = (freetextRequest, typeKey) => {
   const type = typeKey[0];
   const params = {};
-  if (freetextRequest.key) {
+  if (freetextRequest.params.key) {
```

Comment:
This is a fix for `key:value` queries against the view `contacts_by_type_freetext`. Without this, these queries would fail silently and the select2 input would show an infinite loader until a different query would return results. If this has never been noticed before I believe the `key:value` search feature isn't used very much...

Reply:
🤯

Reply:
Thinking about this again, I feel like we really should include a unit test for this case (if possible).